elkscmd/test/fat/qemu.sh

```diff
@@ -1,7 +1,7 @@
-qemu-system-i386 -boot order=ac -fda ../../../elkscmd/full3 -hda ./fat32_hdd
```

Comment:
Can we auto-detect this at script runtime just in case someone's not on x86-64?

Reply:
Yes, sure, I keep this point in my mind.